spl/labels.py

```diff
@@ -186,55 +198,90 @@ def __get_application_numbers(self, set_id, bs_content):
         return list(application_numbers)
 
     def __get_label_text(self, set_id, bs_content):
-        def get_xml_text(navigable_title):
-            # Process the label text
-            label_text = (
-                re.sub(r"\n{3,}", "\n\n", navigable_title.get_text())
-                # Remove initial / trailing white space
-                .lstrip().rstrip()
-            )
-            return label_text
+        def get_xml_text(text):
+            # Process the label text; normalize remove nbsp
```

Comment:
The input to this function is `.text` from the navigable string. Does it handle nested strings? Perhaps under a different tag like `table` ?

That's why the `get_text()` method was being used here.

Reply:
I believe these functions are all the same.  See https://stackoverflow.com/questions/52093875/gettext-vs-text-vs-get-text.  As far I as know text is a property that calls get_text()

Reply:
I see.. Thanks for the clarification.